Assets/AltTester/Editor/AltTesterEditorWindow.cs

```diff
@@ -62,6 +62,9 @@ public class AltTesterEditorWindow : UnityEditor.EditorWindow
         public static UnityEngine.Texture2D PortForwardingTexture;
         public static UnityEngine.Texture2D selectedTestsCountTexture;
 
+        private const string regexForWindows = @"https://altom.com/app/uploads/AltTester/desktop/AltTesterDesktop[\w\.]*.exe";
+        private const string regexForMacOS = @"https://altom.com/app/uploads/AltTester/desktop/AltTesterDesktop[\w\.]*.dmg";
+
         private static string downloadURL;
```

Comment:
the location path is the same. we don't need two variables for that. the only thing that differs is the extension of the file, which can be treated later, when you know the OS (you can add the extension after the regexp). this way, if you need to change the location, you would change it only once.